service/controller/v15/controllercontext/status.go

```diff
@@ -1,17 +1,29 @@
 package controllercontext
 
 type Status struct {
-	// HostedZones information is filled by the hostedzone resouce. This
-	// information is used when creating CloudFormation templates.
-	HostedZones HostedZones
+	// Cluster carries an information between cluster controller resource.
+	Cluster Cluster
+	// Drainer carries an information between drainer controller resource.
+	Drainer Drainer
 }
```

Comment:
Status got scoped between controllers. Later we can even decide to scope it by resources.